packages/core/src/sass/app/_common/components/account-common.scss

```diff
@@ -453,6 +453,22 @@
             padding-bottom: unset;
         }
     }
+
+    &-management {
+        &__message-content {
+            @include mobile {
+                padding-top: 2.4rem;
+            }
+        }
+        &__container {
+            @include desktop {
```

Comment:
Do we need this check

Reply:
The respective width is currently required only for Desktop

Reply:
but then since we dont have tablet view, even if you dont put the check, it would still work as default right 👀